2주차/포켓몬.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+input_list = sys.stdin.readline().strip().split()
+n, m = int(input_list[0]), int(input_list[1])
```

Comment:
```suggestion
n, m = map(int, sys.stdin.readline().strip().split())
```
여기도 마찬가지로 map이랑 언패킹 이용해서 한 줄로 받는 연습 해보세요